src/ontology/ro-edit.owl

```diff
@@ -6147,6 +6149,22 @@ AnnotationAssertion(rdfs:label obo:RO_0016002 "has disease")
 AnnotationAssertion(rdfs:seeAlso obo:RO_0016002 "https://github.com/oborel/obo-relations/issues/478")
 SubObjectPropertyOf(obo:RO_0016002 obo:RO_0016001)
 
+# Object Property: obo:RO_0018001 (is myristoyltransferase activity)
```

Comment:
Is this reflexive? we would want to see something like domains and ranges here. Usually, relations starting with `is` and not ending with `of` are reflexive, i.e. relating an individual to itself. `is person`.

Reply:
This was following the same form of the phosphorylates and ubiqutinates, so this question is also valid for those. I think we need to pull in Chris to reconsider some of these things that got bumped for later

Reply:
If I understand correctly, the helper relation is only there for use in a property chain that defines **myristoylates**.  It sits under:

rdfs:label "helper property (not for use in curation)"
'curator note' "This property or its subproperties is not to be used directly. These properties exist as helper properties that are used to support OWL reasoning."

I think the logic makes sense and the precedent with phosphorylates and ubiquitinates supports adding it.